test/ToastBodySpec.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Toast from '../src/Toast';
+
+describe('Toast.Body', () => {
```

Comment:
Not sure, why e.g. this test fails. Referring documentation: https://airbnb.io/enzyme/docs/api/ShallowWrapper/equals.html